helm/values.yaml

```diff
@@ -202,14 +202,25 @@ fluentbit:
     service: ""
     inputs: |
       [INPUT]
-          Name tail
-          Path /tmp/logs/*.log
-          Tag app.access.log
-          Mem_Buf_Limit 5MB
+          Name            tail
+          Path            /tmp/logs/*.log
+          Tag             app.access.log
+          Mem_Buf_Limit   5MB
           Skip_Long_Lines Off
-    filters: ""
+    filters: |
+      [FILTER]
+          Name  Lua
+          Match *
+          call  append_tag
+          code  function append_tag(tag, timestamp, record) new_record = record new_record["datestamp"] = os.date("%Y%m%d", timestamp) return 1, timestamp, new_record end
```

Comment:
cool - Lua code!

seems to be appending a tag to each record containing a datestamp.

Reply:
That's right. It's pretty hacky but I'm just wanting to finish this thing up even if it isn't done in the best way.